src/Gateways/AbstractGateway.php

```diff
@@ -625,7 +625,7 @@ protected function submit_request( Requests\RequestInterface $request ) {
 	 * @return bool
 	 */
 	protected function handle_response( Requests\RequestInterface $request, Transaction $response ) {
-		if ($response->responseCode !== '00' && 'SUCCESS' !== $response->responseCode || $response->responseMessage === 'Partially Approved') {
+	    if ($response->responseCode !== '00' && 'SUCCESS' !== $response->responseMessage || $response->responseMessage === 'Partially Approved') {
```

Comment:
I believe the comparison of `'SUCCESS'` and `$response->responseCode` is what we need for GP API. @apetrovici Are you able to confirm?

Reply:
@slogsdon yes, that is for GP-API

Reply:
Thanks!

Reply:
ok.. reverted the changes for result code